intermine_builder/build.sh

```diff
@@ -108,7 +108,7 @@ if [ ! -f /home/intermine/intermine/${MINE_NAME:-biotestmine}/project.xml ]; the
         cp /home/intermine/intermine/configs/project.xml /home/intermine/intermine/${MINE_NAME:-biotestmine}/
         echo "$(date +%Y/%m/%d-%H:%M) Set correct source path in project.xml"
         sed -i 's/'${IM_DATA_DIR:-DATA_DIR}'/\/home\/intermine\/intermine\/data/g' /home/intermine/intermine/${MINE_NAME:-biotestmine}/project.xml
-        sed -i 's/dump="true"/dump="flase"/g' /home/intermine/intermine/${MINE_NAME:-biotestmine}/project.xml     
+        sed -i 's/dump="true"/dump="flase"/g' /home/intermine/intermine/${MINE_NAME:-biotestmine}/project.xml
```

Comment:
I know you didn't edit this line, but I noticed there's a typo "flase" instead of "false". Could you fix this now in case it rears its head as a different bug in the future?

Reply:
oh, right. I'll update that.

Reply:
done. Strangely, the same type was there in two other places.